core/storage/file/gae_models_test.py

```diff
@@ -0,0 +1,197 @@
+            file_models.FileMetadataModel._construct_id( # pylint: disable=protected-access
+                'exp_id1', 'path/to/file.png'))
+        file_metadata_model_id2 = (
+            file_models.FileMetadataModel._construct_id( # pylint: disable=protected-access
+                'exp_id1', '/exp_id1/path/to/file.png'))
+
+        self.assertEqual(
+            file_metadata_model_id1, '/exp_id1/path/to/file.png')
+        self.assertEqual(
+            file_metadata_model_id2, '/exp_id1/path/to/file.png')
+
+    def test_create(self):
+        file_metadata_model = file_models.FileMetadataModel.create(
+            'exp_id1', 'path/to/file.png')
+        self.assertEqual(file_metadata_model.deleted, False)
+        self.assertEqual(
+            file_metadata_model.id, '/exp_id1/path/to/file.png')
+
+    def test_get_model(self):
+        file_metadata_model = file_models.FileMetadataModel.create(
+            'exp_id1', 'path/to/file1.png')
+        file_metadata_model.commit(feconf.SYSTEM_COMMITTER_ID, [])
+        actual_model = file_models.FileMetadataModel.get_model(
+            'exp_id1', 'path/to/file1.png')
+
+        self.assertEqual(actual_model, file_metadata_model)
+
+        actual_model = file_models.FileMetadataModel.get_model(
+            'exp_id1', 'path/to/file2.png')
+        self.assertEqual(actual_model, None)
+
+        with self.assertRaisesRegexp(Exception, (
+            'Entity for class FileMetadataModel '
+            'with id /exp_id1/path/to/file2.png not found')):
+            actual_model = file_models.FileMetadataModel.get_model(
+                'exp_id1', 'path/to/file2.png', True)
+
+    def test_get_version(self):
+        file_metadata_model = file_models.FileMetadataModel.create(
+            'exp_id1', 'path/to/file1.png')
+        file_metadata_model.commit(feconf.SYSTEM_COMMITTER_ID, [])
+
+        actual_model = file_models.FileMetadataModel.get_version(
+            'exp_id1', 'path/to/file1.png', 1)
+        self.assertEqual(file_metadata_model.key, actual_model.key)
+
+        file_metadata_model.commit(feconf.SYSTEM_COMMITTER_ID, [])
+        actual_model = file_models.FileMetadataModel.get_version(
+            'exp_id1', 'path/to/file1.png', 2)
+        self.assertEqual(file_metadata_model.key, actual_model.key)
+
+    def test_commit(self):
+        file_metadata_model = file_models.FileMetadataModel.create(
+            'exp_id1', 'path/to/file1.png')
+        self.assertEqual(file_metadata_model.version, 0)
+
+        file_metadata_model.commit(feconf.SYSTEM_COMMITTER_ID, [])
+        self.assertEqual(file_metadata_model.version, 1)
+
+
+class FileModelTest(test_utils.GenericTestBase):
+    """Tests the FileModel class."""
+
+    def test_reconstitute(self):
+        file_model = file_models.FileModel.create(
+            'exp_id1', 'path/to/file.png')
+        self.assertEqual(file_model.content, None)
+        file_model._reconstitute( # pylint: disable=protected-access
```

Comment:
I think in most of these places, the code will fit in a single line, and then you can add the pylint pragma (I don't think it enforces the 80 char line limit when you use the pragma.

If so, could you make all such occurrences a single line?

Reply:
Done.